2019 3 13 Fixed Python return types

fkong7 opened the pull request.

osmsc commented:
@fkong7 - Fanwei - can you explain why all the return types have been updated?  This seems like a big change.  Is this to fix a bug or something?

fkong7 commented:
The Python returns previously followed the TCL bindings to return strings for most of the cases. For example, a bool was returned as a string of 1 or 0; a point list was returned as a string rather than as a Python list. This seemed to cause inconvenience when people want to directly use the return values (such as using the bool for a if statement). I fixed this by changing the return types to what make more sense for Python. Also some of the functions returned "success" before and I changed it to return true or false. This way people can check the return directly in Python without having to compare it with a "success" string.
